apps/web/src/pages/templates/components/ListProviders.tsx

```diff
@@ -3,16 +3,18 @@ import { ChannelTypeEnum } from '@novu/shared';
 
 import { When } from '../../../components/utils/When';
 import { Button, colors } from '../../../design-system';
-import { IIntegratedProvider } from '../../integrations/IntegrationsStorePage';
+import type { IIntegratedProvider } from '../../integrations/types';
 import { stepNames } from '../constants';
 import { ChannelTitle } from './ChannelTitle';
 import { LackIntegrationError } from './LackIntegrationError';
 
 export const ListProviders = ({
+  channel,
   providers,
   setConfigureChannel,
   setProvider,
 }: {
+  channel: ChannelTypeEnum;
```

Comment:
changed to pass the channel directly than having the logic to take the first el and take the channel from there, which was breaking